contracts/src/v0.8/automation/ZKSyncAutomationForwarder.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: BUSL-1.1
-pragma solidity ^0.8.16;
+pragma solidity ^0.8.19;
```

Comment:
any reason to not lock this to `0.8.19` (instead of `^0.8.19`) I see we do that in other registry contracts

Reply:
no real reasons. i lock it to 0.8.19